newIDE/app/src/MainFrame/EditorContainers/HomePage/GetStartedSection/Questionnaire.js

```diff
@@ -0,0 +1,337 @@
+    ],
+  },
+  workingTeam: {
+    text: t`How are you working on your projects?`,
+    nextQuestion: 'painPoints',
+    answers: [
+      {
+        text: t`Completely alone`,
+        id: 'alone',
+        imageSource: 'res/questionnaire/completely-alone.svg',
+      },
+      {
+        text: t`With at least one other person`,
+        id: 'onePlus',
+        imageSource: 'res/questionnaire/with-at-least-one-other-person.svg',
+      },
+      {
+        text: t`With an established team of people during the whole project`,
+        id: 'team',
+        imageSource: 'res/questionnaire/with-a-team.svg',
+      },
+    ],
+  },
+  painPoints: {
+    text: t`Is there anything that you struggle with while working on your projects?`,
+    multi: true,
+    nextQuestion: 'targetDate',
+    answers: [
+      {
+        text: t`Lack of Graphics & Animation`,
+        id: 'lackGraphics',
+        imageSource: 'res/questionnaire/lack-of-graphics.svg',
+      },
+      {
+        text: t`Lack of Music & Sound`,
+        id: 'lackSound',
+        imageSource: 'res/questionnaire/lack-of-sounds.svg',
+      },
+      {
+        text: t`Lack of Marketing & Publicity`,
+        id: 'lackMarketing',
+        imageSource: 'res/questionnaire/lack-of-marketing.svg',
+      },
+      {
+        text: t`Implementing in-project monetization`,
+        id: 'inAppMonetization',
+        imageSource: 'res/questionnaire/in-app-monetization.svg',
+      },
+      {
+        text: t`Dealing with data integration from external sources`,
+        id: 'externalIntegration',
+        imageSource: 'res/questionnaire/integration-of-external-services.svg',
+      },
+      {
+        text: t`Other`,
+        id: 'input',
+        imageSource: 'res/questionnaire/other.svg',
+        isFree: true,
+      },
+    ],
+  },
+  targetDate: {
+    text: t`When do you plan to finish or release your projects?`,
+    nextQuestion: 'gameDevelopmentExperience',
+    answers: [
+      {
+        text: t`Less than a month`,
+        id: '1MonthOrLess',
+        imageSource: 'res/questionnaire/delay-less-than-1-month.svg',
+      },
+      {
+        text: t`Around 1 or 2 months`,
+        id: '1To2Months',
+        imageSource: 'res/questionnaire/delay-1-to-2-months.svg',
+      },
+      {
+        text: t`Around 3 to 5 months`,
+        id: '3To5Months',
+        imageSource: 'res/questionnaire/delay-3-to-5-months.svg',
+      },
+      {
+        text: t`More than 6 months`,
+        id: '6MonthsPlus',
+        imageSource: 'res/questionnaire/delay-6-months-plus.svg',
+      },
+      {
+        text: t`In around a year`,
+        id: '1Year',
+        imageSource: 'res/questionnaire/delay-1-year.svg',
+      },
+      {
+        text: t`I don’t have a specific deadline`,
+        id: 'noDeadline',
+        imageSource: 'res/questionnaire/delay-no-deadline.svg',
+      },
+    ],
+  },
+  gameDevelopmentExperience: {
+    text: t`What kind of projects do you want to build with GDevelop?`,
```

Comment:
question seems wrong here

Reply:
Good catch! Weird I didn't see it before